### PR TITLE
fix(runtime/ops/signal.rs): Add FreeBSD signal definition.

### PR DESCRIPTION
**Pull request by MikaelUrankar**

### CODE REVIEW

File: runtime/ops/signal.rs [hunk 1]
```diff
@@ -57,6 +57,44 @@ impl Resource for SignalStreamResource {
...
+    "SIGIO" => Ok(23),
+    "SIGXCPU" => Ok(24),
+    "SIGXFSZ" => Ok(25),
+    "SIGVTALRM" => Ok(26),
+    "SIGPROF" => Ok(27),
+    "SIGWINCH" => Ok(28),
+    "SIGINFO" => Ok(29),
+    "SIGUSR1" => Ok(30),
+    "SIGUSR2" => Ok(31),
+    _ => Err(type_error(format!("Invalid signal : {}", s))),
```
Comment (lucacasonato) [inline runtime/ops/signal.rs:94]: According to https://www.freebsd.org/cgi/man.cgi?sektion=3&query=signal there are also `SIGTHR` and `SIGLIBRT`. Do you want to add those too?

**Review by lucacasonato — COMMENTED** [review]
Thanks. We don't officially support FreeBSD, but I am happy to merge this if this unblocks your build :-)

File: runtime/ops/signal.rs (hunk 1 above)
Comment (MikaelUrankar) [reply runtime/ops/signal.rs:96]: I've added the missing signal. Thanks.

**Review by lucacasonato — APPROVED** [review]
Thanks @MikaelUrankar

**Pull request merged**